Cargo.toml

```diff
@@ -43,8 +43,8 @@ sea-query-derive = { version = "^0.2.0", path = "sea-query-derive", optional = t
 sea-query-driver = { version = "^0.2.0", path = "sea-query-driver", optional = true }
 serde_json = { version = "^1", optional = true }
 bytes = { version = "^1", optional = true }
-chrono = { version = "^0", default-features = false, features = ["clock"], optional = true }
-postgres-types = { version = "^0", optional = true }
+chrono = { version = "0.4", default-features = false, features = ["clock"], optional = true }
+postgres-types = { version = "0.2", optional = true }
```

Comment:
Hey @Dylan-DPC, thanks for pointing this out!

I think we can relax it to be `^0.x`. That way, we're always on the latest available patch version.

Reply:
Cargo does ^ by default so this is the same thing just implicit